src/documents/CoreAPI.md

```diff
@@ -539,6 +539,22 @@ if you are interested.
 path segments representing this
 * `newValue: {Any}` The value to be written into `model` at the path `path`
 
+### fluid.modelPairToChanges(value, oldValue, changePathPrefix)
+
+* `value {Any}` Model value to compare
+* `oldValue {Any}` Model value to compare
+* `changePathPrefix {String|Array of String}` [optional] Path prefix to prepend to change record paths
+* Returns: `{Array of Object}` An array of change records
+
+Calculates the changes between the model values `value` and `oldValue`
+and returns an array of change records. The optional argument
+`changePathPrefix` is prepended to the change path of each record (this
+is useful for generating change records to be applied at a non-root path
+in a model).
```

Comment:
It would be really useful to have a super-simple example here that might allow people to get a sense of what is going on - for example, a computation of the changes between {a: 1} and {b:2}